omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs1_8/ConceptSearchHandler1_8.java

```diff
@@ -53,7 +54,9 @@ public class ConceptSearchHandler1_8 implements SearchHandler {
 	            new SearchQuery.Builder("Allows you to find concepts by source and code").withRequiredParameters("source")
 	                    .withOptionalParameters("code").build(), new SearchQuery.Builder(
 	                    "Allows you to find concepts by name and class").withRequiredParameters("name")
-	                    .withOptionalParameters("class", "searchType").build()));
+	                    .withOptionalParameters("class", "searchType").build(), new SearchQuery.Builder(
+	                    		"Allows you to find a list of concepts by passing concept uuids")
+	                    .withRequiredParameters("references").build()));
```

Comment:
Is `references` supposed to mean `uuid` and `mappings`?

Reply:
Are we intentionally making `references` a required parameter?

Reply:
> Is `references` supposed to mean `uuid` and `mappings`?

Actually `references` is meaning either `uuids` or `mapping`.

Reply:
> Is `references` supposed to mean uuid and mappings?

Technically, `references` is taken from the `getConceptByReference()` function for 2.6. `getConceptByReference()` has two other pathways that don't seem as relevant: one to load a constant from Java code via reflection with seems potentially dangerous to expose via REST and one to get concepts by the database id, which is generally not exposed through the REST API anyway. We could call it `mappingOrConceptUuid` but that seems a little verbose.

> Are we intentionally making references a required parameter?

Well, we've got three different supported queries here. The one that takes the `references` parameter doesn't accept any of the other parameters, so it's not exactly optional..

Reply:
Makes sense! 😊